src/api/app/views/webui2/webui/user/_edit_dialog.html.haml

```diff
@@ -0,0 +1,16 @@
+.modal.fade#edit-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'branch-modal-label', hidden: true } }
+  .modal-dialog.modal-dialog-centered{ role: 'document' }
+    .modal-content
+      .modal-header
+        %h5.modal-title#branch-modal-label Change Your Password
+      = form_for(user, url: user_save_path, method: 'post') do |f|
+        .modal-body
+          = f.hidden_field(:login, id: 'user')
+          .name
+            = f.label(:realname, 'Name:')
```

Comment:
@Ana06, I think you are missing some classes. This should be something like:

```
 .form-group
   = f.label(:realname, 'Name:')
   = f.text_field(:realname, class: 'form-control')
```

Reply:
why is it not being followed? It seems to me like our pattern is wrong. It is not following Bootstrap...

Reply:
There is nothing about text fields in the pattern library :confused:

Reply:
@Ana06 and I cleared things up. It was marked as resolved, but it wasn't pushed yet. It's now all good.

Reply:
Is it the reason why the input fields are not aligned? Because that looks not nice :thinking: 

![selection_012](https://user-images.githubusercontent.com/3799140/49143277-020e3a00-f2fb-11e8-9014-ade8bfd83252.png)